tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/Makefile.inc

```diff
@@ -0,0 +1,6 @@
+ifeq ($(TARGET), zephyr_vexriscv)
+	export ZEPHYR_TOOLCHAIN_VARIANT?=zephyr
+micro_speech_bin:
+	cmake -B ${GENDIR}/micro_speech/CMake -GNinja -DBOARD="litex_vexriscv" -S tensorflow/lite/micro/examples/micro_speech/zephyr_riscv
```

Comment:
Sounds good to me, we can do that.

Reply:
So as I can read in the CMake docs the option does as follows.

> Enable developer warnings.
>Enable warnings that are meant for the author of the CMakeLists.txt files. By default, this will also turn on deprecation warnings.

I have nothing against doing this.

Reply:
Expect an update in an hour.